File: python/nivel 4 trampas/25.py
"""
Ejercicio 25
Calcular el factorial de un número (iterativo)
"""
numero = int(input("Ingrese un numero: "))

if numero < 0:
  print("El factorial no esta definicio para nuneros negativos")
else:
  factorial = 1

  for i in range(1, numero + 1):
    factorial *= i

  print(f"El factorial de {numero} es: {factorial}")

File: javascript/nivel 4 trampas/25.js
/*
Ejercicio 25
Calcular el factorial de un número (iterativo)
*/

let numero = parseInt(prompt("ingresa un numero: "));

if(numero < 0){
  console.log("El numero facotorial no esta definidio para 0 o numeros negativos");
}else{
  let facotorial =1;

  for(let i = 1; i <=numero; i++){
    facotorial = facotorial * i;
  }
  console.log("el factorial de ",numero,"es: ",facotorial);
}